src/config/schema.ts

```diff
@@ -342,6 +357,16 @@ export const configSchema = {
           duration: 3600,
           blockDuration: 86400,
         },
+        createDeviceAccountPerIp: {
+          points: 4,
+          duration: 86400,
+          blockDuration: 86400 * 10,
+        },
+        createDeviceAccountGlobal: {
+          points: 180,
```

Comment:
```suggestion
          points: 500,
```

I think this value can be higher now that Appcheck has been integrated, it was set pre-Appcheck capability in mind